dom-manipulation/main.js

```diff
@@ -0,0 +1,22 @@
+let count = 0;
+
+const $hotButton = document.querySelector('.hot-button');
+const $clickCount = document.querySelector('.click-count');
+
+$hotButton.addEventListener('click', function (event) {
+  count++;
+  $clickCount.textContent = `Clicks: ${count}`;
+  if (count <= 4) {
+    $hotButton.className = 'hot-button cold';
+  } else if (count <= 7) {
+    $hotButton.className = 'hot-button cool';
+  } else if (count <= 10) {
+    $hotButton.className = 'hot-button tepid';
+  } else if (count <= 13) {
+    $hotButton.className = 'hot-button warm';
+  } else if (count <= 16) {
+    $hotButton.className = 'hot-button hot';
```

Comment:
These should just be `less than` rather than `less than or equal to`

Reply:
![Screenshot 2023-11-06 at 15 50 42](https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/f7835886-8242-4cb1-8566-708d0836a79b)
 
Thanks Shawn! repushed exercise and updated screenshot attached.